packages/headless/src/api/commerce/common/response.ts

```diff
@@ -7,11 +7,18 @@ import {ProductRecommendation} from '../../search/search/product-recommendation'
 import {Pagination} from './pagination';
 import {Sort} from './sort';
 
+export interface RecommendationCommerceSuccessResponse {
+  responseId: string;
+  headline: string;
+  products: ProductRecommendation[];
```

Comment:
Depending on the outcome of this PR / timing when it gets merged, there will be a conflict on this precise type: 

https://github.com/coveo/ui-kit/pull/3738